database/store.go

```diff
@@ -114,21 +108,23 @@ func (s store) GetEvents() ([]model.Event, error) {
 	s.m.Lock()
 	defer s.m.Unlock()
 
-	rows, err := s.db.Query("SELECT * FROM event")
-	if err != nil{
+	rows, err := s.db.Query(QUERY_EVENT_TABLE)
+	if err != nil {
 		return []model.Event{}, err
 	}
 	var events []model.Event
 	for rows.Next() {
-		var tag 				string
-		var name 				string
-		var frontends 			string
-		var exercises 			string
-		var available 			uint
-		var capacity 			uint
-		var startedAt			string
-		var expectedFinishTime 	string
-		var finishedAt			string
+
+		// todo  : optimize this one, looks ugly
+		var tag string
+		var name string
+		var frontends string
+		var exercises string
+		var available uint
+		var capacity uint
+		var startedAt string
+		var expectedFinishTime string
+		var finishedAt string
```

Comment:
Looks ugly there could be better way of handling this one.

Reply:
maybe like 
```golang
var tag,name,frontend,e..... string
```
could be a nicer way